scripts/integration-test

```diff
@@ -47,4 +47,9 @@ while ! kubectl rollout status -w -n cattle-system deploy/rancher-webhook; do
     sleep 2
 done
 
-./bin/rancher-webhook-integration.test -test.v
+./bin/rancher-webhook-integration.test -test.v -test.run IntegrationTest
+
+kubectl scale deploy rancher-webhook -n cattle-system --replicas=0
+sleep 1
```

Comment:
Nit:

You can wait for the webhook to be fully down by using `kubectl wait`, which is a bit more of a "sure thing" than this sleep command.

This command should do the trick:

```bash
kubectl wait pods -l app=rancher-webhook --for=delete -n cattle-system
```

Reply:
Thank you. I couldn't remember what to wait on. I tried looking at the deployment status, but that returned before this.